src/ui/sass/modules/_SearchBar.scss

```diff
@@ -216,6 +216,11 @@ $searchbar-button-text-color-active: var(--yxt-searchbar-button-text-color-base)
     transform-origin: 50% 50%;
     animation: dash 1.4s ease-in-out infinite;
   }
+
+  &-Icon--inactive &-LoadingIndicator-AnimatedIcon 
```

Comment:
I think if you just remove line 187-192 (which was used to keep the animation running while not on display), the display: none should work on the loading indicator without additional scss

Reply:
If we keep the PR as-is, the starting position of the animation will actually change as if the animation was running the whole time. If we delete those lines then the loading icon will always start from the same spot